README.md

```diff
@@ -217,7 +228,27 @@ You create your own Crashes listener and assign it like this:
         }
         ```
 
-## 6. Advanced APIs
+## 6. Distribute APIs
+
+You can easily let your users to get the latest version of your app by integrating `Distribute` module of Mobile Center SDK. All you need to do is pass the module name as a parameter in the `Start()` API call. Once the activity is created, the SDK checks for new updates in the background. If it finds a new update, users will see a dialog with two options - `Download` and `Ignore`. If the user presses `Download`, it will trigger the new version to be installed.
```

Comment:
users to get => users getting

Also, I thought we were using `service` and not `module` or `feature`.